packages/api-db-mongodb/src/db/article.ts

```diff
@@ -285,34 +285,34 @@ export class MongoDBArticleAdapter implements DBArticleAdapter {
         }
       : {}
 
-    let stateFilter: FilterQuery<any> = {}
-    let textFilter: FilterQuery<any> = {}
+    const stateFilter: FilterQuery<any> = {}
```

Comment:
thanks for improving the code. But in general please open a new PR for those kind of changes. It's hard for the reviewer to see what are just linting changes and what changes functionality.

Reply:
Alright  Agree, It's the auto linting.